Batch File/Batch File.sublime-syntax

```diff
@@ -34,6 +35,12 @@ contexts:
     - match: '<&?|>[&>]?'
       scope: keyword.operator.redirection.dosbatch
 
+  labels:
+    - match: '(?i):(\w+)\r?\n'
+      captures:
+        0: meta.function.dosbatch
+        1: entity.name.function.dosbatch
```

Comment:
For consistency with other syntaxes, this should be:

```yaml
  labels:
    - match: '^\s*(:)([\w ]+)$
      captures:
        1: punctuation.separator.dosbatch
        2: entity.name.label.dosbatch
```

A meta scope shouldn't really be necessary since it is such a simple construct.

According to https://technet.microsoft.com/en-us/library/bb490914.aspx, a label can contain spaces and must be on a line by itself.

A `.tmPreferences` file should probably also be added to show `entity.name.label` in the symbol list.